package/src/components/AttachmentPicker/components/AttachmentPickerBottomSheetHandle.tsx

```diff
@@ -20,20 +18,14 @@ const styles = StyleSheet.create({
 export const AttachmentPickerBottomSheetHandle: React.FC<{
   animatedIndex: Animated.SharedValue<number>;
 }> = ({ animatedIndex }) => {
-  const {
```

Comment:
is it better to do removals or comments? I'm leary of these coming back in on an upgrade...